modules/mbedtls/CMakeLists.txt

```
@@ -8,6 +8,7 @@ if(CONFIG_MBEDTLS_BUILTIN)
 
   target_include_directories(mbedTLS INTERFACE
     ${ZEPHYR_CURRENT_MODULE_DIR}/include
+    ${ZEPHYR_CURRENT_MODULE_DIR}/library
```

This is the commit message:
> psa_crypto_cipher.c includes an internal header with <>, causing complication errors in Zephyr (header not found).

is this something that is considered an error in mbedTLS itself ?
So that we can expect this to be cleaned up in a later release.

If that's the case, a comment here, similar to this:
```suggestion
    # mbedTLS v3.1.0 has psa_crypto_cipher.c including an internal header using <>.
    # This line ensure the header can be found in Zephyr.
    # When updating to mbedTLS > v3.1.0, please check if this has been corrected, and if som remove this include path.
    ${ZEPHYR_CURRENT_MODULE_DIR}/library
```

---

I've added the comment.